src/drivers/wayland/wayland.h

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file wayland
+ *
+ */
+
+#ifndef WAYLAND_H
+#define WAYLAND_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../../lvgl.h"
+#if LV_USE_WAYLAND
```

Comment:
You need to run `lv_conf_internal_gen.py` from the script folder.

Reply:
> You need to run `lv_conf_internal_gen.py` from the script folder.

thanks...